integration-tests/src/test/resources/exporter/createGrafana.sh

```diff
@@ -5,23 +5,57 @@ monitoringExporterDir=$1
 resourceExporterDir=$2
 grafanaVersionArgs=$3
 monitoringExporterEndToEndDir=${monitoringExporterDir}/src/samples/kubernetes/end2end
-
+CHARTNS=monitortestns
+CHARTNAME=grafana
 HELM_VERSION=$(helm version --short --client)
 
+function checkPod() {
+  max=20
+  count=0
+  echo "Checking pods $CHARTNAME"
+  grafanapod=$(kubectl get po -n $CHARTNS | grep $CHARTNAME | awk '{print $1 }')
```

Comment:
At this point you are sure that the grafana pod at least created ?

Reply:
the grafanapod value is used only for debugging of failure. There is another check for helm install success before calling this code. So at this point pod is created but might has failures